Add js for drop-down menu

SimonGrchevski opened the pull request:
- Add the javascript logic for displaying menu on click for 2 different screen sizes
- Add the javascript logic for closing the menu bar when clicked on X or in some of the li's
- Add styling to the drop-down menu
- Position the closing tag in the right corner
- Add space between the menu items
- Position the menu items to the left corner

TouringIdk committed:
Add menu display

SimonGrchevski committed:
Add event listener

TouringIdk committed:
Add Linters

TouringIdk committed:
Merge branch 'menu-event' of https://github.com/SimonGrchevski/Portfolio into menu-event

SimonGrchevski committed:
Fix EsLinter errors

SimonGrchevski committed:
Fix indentation errors

SimonGrchevski committed:
Add arrow function

SimonGrchevski committed:
Add empty line at the end of js file

SimonGrchevski committed:
Fix the extranious closing script

TouringIdk committed:
Add functions to the menu

SimonGrchevski committed:
Position the closing button

TouringIdk committed:
Change the X color to grey

TouringIdk committed:
Abs dissapearing I

SimonGrchevski committed:
Remove the background when the menu is oppened

TouringIdk committed:
Add welcome conditionals

TouringIdk committed:
Fix Linter Errors I

TouringIdk committed:
Fix Linter Errors II

TouringIdk committed:
Fix Linter Errors III

TouringIdk committed:
Fix Linter Errors IV

SimonGrchevski committed:
Remove the logo in desktop mode

TouringIdk committed:
Fix Linter Error V

TouringIdk committed:
Fix Linter Error VI

TouringIdk committed:
Fix Linter Error VII

TouringIdk committed:
Finally Fix All Linter Errors

Temmarie commented on index.html:
```diff
@@ -23,12 +23,12 @@
+      <ul class="menu-list" id="menu-list">
+        <li><a href="#works" onclick="show()">Portfolio</a></li>
+        <li><a href="#about-me" onclick="show()">About</a></li>
+        <li><a href="#contact" onclick="show()">Contact</a></li>
```
- [ ] Good job adding the functions that enable the drop-down menu to work!
However, there are a few changes required for you to make here: 
- The nav items, social media icons and links on the page are all missing their required styles. Kindly add them back so they match with the Figma design template.
- The drop-down menu should cover the entire page.
- You are missing a background image decoration in the headline. Kindly add it.
_See image below for reference_

**Original** | **Your clone**
  ----------- | -----------
 ![image](https://user-images.githubusercontent.com/49621279/122804655-044f8f80-d27d-11eb-9359-8a4c5df99e7d.png) | ![image](https://user-images.githubusercontent.com/49621279/122804705-13364200-d27d-11eb-8713-8a127b9cd6e1.png) 
![image](https://user-images.githubusercontent.com/49621279/122804977-5f818200-d27d-11eb-99a6-50b3ac09fa47.png) | ![image](https://user-images.githubusercontent.com/49621279/122805019-6c05da80-d27d-11eb-8603-569d12f3b44b.png)
![image](https://user-images.githubusercontent.com/49621279/122804448-ca7e8900-d27c-11eb-9e6b-9b8c3d4c0a21.png) | ![image](https://user-images.githubusercontent.com/49621279/122804539-e4b86700-d27c-11eb-9778-c62800c661ba.png)
![image](https://user-images.githubusercontent.com/49621279/122805213-aa9b9500-d27d-11eb-925d-2cd1a8e9cefa.png) | ![image](https://user-images.githubusercontent.com/49621279/122805415-ec2c4000-d27d-11eb-87e5-9e28113a7412.png)

Davidosky007 replied:
* As the previous reviewer pointed out your design social media icon links still look different 
Yours:
![Screenshot (92)](https://user-images.githubusercontent.com/52497246/122809918-a2b41300-d2c6-11eb-8260-782fc64d3924.png)
____________________________________________________________________________________________________________________________

![Screenshot (90)](https://user-images.githubusercontent.com/52497246/122810091-d8f19280-d2c6-11eb-9f54-c50c90647eb5.png)
________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________
Figma:
![Screenshot (91)](https://user-images.githubusercontent.com/52497246/122810220-00485f80-d2c7-11eb-8995-6a05852fa70c.png)

![Screenshot (89)](https://user-images.githubusercontent.com/52497246/122810249-09d1c780-d2c7-11eb-8aad-7a2424ac2d72.png)

Temmarie reviewed (changes requested):
## STATUS: CHANGES REQUIRED! :recycle:

### Hi @SimonGrchevski     ! :wave: :wave:

#### You have done really well in implementing the requirements.
- You used proper GitHub Flow ✔️ 
- Your linter checks are all passing ✔️ 
- Your README and PR message are descriptive ✔️ 
- You have successfully implemented the dropdown menu ✔️  

 However, there are changes required for you to make in order for this project to be approved. 

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. :smile:

- - -

### **Required Changes** ♻️
- [ ] Kindly note that you have to write a descriptive Pull Request Message with a descriptive title as well. It's essential that your repository is professional, as this is one of the things that potential colleagues will look at when recruiting team members. 

_See the inline comments below for more required changes_


### Optional Changes 👌
Every comment with the OPTIONAL prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend that you take them into account as they can make your code better.
- N/A

_If anything doesn't seem clear to you, feel free to leave your question or comment in the PR thread below_

**_"Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise."_**
- - - 
**Note that you still have 2 more reviews as written in the [Code Reviews Limit Policy](https://microverse.zendesk.com/hc/en-us/restricted?return_to=https%3A%2F%2Fmicroverse.zendesk.com%2Fhc%2Fen-us%2Farticles%2F1500004088561), however, if you feel this review is unfair, you can request a second opinion using [this](https://airtable.com/shrQAqnBwek5a0O0s) form.**

__Kindly make the necessary changes and submit another code review request.__

SimonGrchevski committed:
Fix menu-items font size and padding

SimonGrchevski committed:
Merge branch 'menu-event' of github.com:SimonGrchevski/Portfolio into menu-event

Davidosky007 reviewed (changes requested):
Hi @SimonGrchevski @eroiyo, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

SimonGrchevski committed:
Fix anchor's color

SimonGrchevski committed:
Edit the request changes for new CR

SimonGrchevski committed:
Fix Linter Errors for colour

SimonGrchevski committed:
Add styling and positioning to the hr when the drop down menu is opened

SimonGrchevski committed:
Fix the whitespace error on line 31

SimonGrchevski committed:
Add mobile-nav styles

SimonGrchevski committed:
God help me

SimonGrchevski committed:
Add the fixed header

SimonGrchevski committed:
Rewind the errors

SimonGrchevski committed:
Fix the headline padding and button padding

SimonGrchevski committed:
Add the skills images

TouringIdk committed:
Add Cristal of the About Section

TouringIdk committed:
Add last cristal for the about seccion

SimonGrchevski committed:
Add horizontal padding to cards, fix the buttons padding

SimonGrchevski committed:
Fix desktop sizes paddings and positioning on cards

SimonGrchevski committed:
Align items in form container to center

SimonGrchevski committed:
Fix social li in the footer

SimonGrchevski committed:
Add finishing touch

Add footer with social icons

Position the footer

Add bacground to the header

Expend the header to the whole width

Add sticky positioning to the header

Fix JS dropdown bug

Position the closing tag of the dropdown

Remove the padding from drop-down list

Add correct font-size to the drop-down list

Make the mobile menu fill the whole screen

SimonGrchevski committed:
Remove commented code

SimonGrchevski committed:
Fix Empty spaces in css

SimonGrchevski committed:
Fix indentation LinterErrors

SimonGrchevski committed:
Add more description to the README.md

SimonGrchevski commented:
Changes made:
  - The drop-down menu is filling the whole site
  - Add the correct colour for the links
  - Add right font-size to the links
  - Add background to the headline and remove the previous
  - Add a footer with the social links
  - Refactor the HTML structure
  - Add the missing footer
  - Add background to the header
  - Make navigation bar fixed

smcommits reviewed (approved):
### Status: Approved 👍🏽 

Hi @SimonGrchevski, 

## You have done a great job! Your project is complete. This PR is now approved. You can go ahead and merge the branch. Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
